src/components/screen/MemoList/MemoList.tsx

```diff
@@ -0,0 +1,23 @@
+import { MemoListProps } from './MemoList.types'
+
+const MemoList: React.FC<MemoListProps> = ({ users }) => {
+  const filterUsers = users.filter((user) => {
+    return user.memo != '' && user.memo
+  })
+  return (
+    <div className='w-8/12 rounded-lg bg-white p-8'>
+      <div className='grid grid-cols-4 gap-5'>
+        <p className='mb-8 text-center text-3xl underline'>メモ欄</p>
+        <div className='col-span-3'></div>
+        {filterUsers.map((user) => (
+          <>
+            <p className='text-center text-xl'>{user.name}</p>
+            <p className='col-span-3 my-auto'>{user.memo}</p>
+          </>
+        ))}
+      </div>
+    </div>
```

Comment:
colを埋めるためのdiv要素を作るんだったら、タイトルはgridの外に出してしまっていいと思う

あと要素と要素の隙間を開ける時はmbじゃなくてflexかgridのgapを使うようにしよう
今回はgapを当ててたけど、タイトルだけもう少し隙間を開けたかったからmbを振ったと思うんだけど、それならそもそもgirdから外した方がいいかなという感じ！

今回は、タイトルとメモの空間が寂しいから、daisyUIのdividerあたりで埋めるのがいいと思う
あと名前ととメモの羅列が質素になってしまうから、名前にbadgeとか当てると可愛いかもね

```suggestion
    <div className='w-2/3 rounded-lg bg-white p-8'>
      <p className='text-3xl'>メモ欄</p>
      <div className='divider' />
      <div className='grid grid-cols-4 gap-5'>
        {filterUsers.map((user) => (
          <>
            <p className='mx-auto badge badge-lg'>{user.name}</p>
            <p className='col-span-3 my-auto'>{user.memo}</p>
          </>
        ))}
      </div>
    </div>
```